### PR TITLE
Fpg 31 - Add health to player

### PR DESCRIPTION
**Pull request by dprzygocka**

### CODE REVIEW

File: public/models/HealGem.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    onCollect(player) {
+      this.healPlayer(player);
+    }
+  
+    //decide whether to heal current player or other players
+    healPlayer(player) {
+      if (this.affectsMe) {
+        this.heal(player);
+      } else {
+        socket.emit('healPlayer');
```
Comment (mana998) [inline public/models/HealGem.js:22]: I think you still need to add this to app.js

Comment (dprzygocka) [reply public/models/HealGem.js:22]: you mean the socket?

Comment (mana998) [reply public/models/HealGem.js:22]: yes cause you emit it to server but you don't handle it on the server

Comment (dprzygocka) [reply public/models/HealGem.js:22]: okok I see now

**Pull request merged**